dandischema/models.py

```diff
@@ -1605,14 +1624,20 @@ def contributor_musthave_contact(
 
     id: str = Field(
         description="Uniform resource identifier",
-        pattern=r"^(dandi|DANDI):\d{6}(/(draft|\d+\.\d+\.\d+))$",
+        # Vendored:
+        # pattern=r"^(dandi|DANDI):\d{6}(/(draft|\d+\.\d+\.\d+))$",
+        # Unvendored:
+        pattern=r"^([a-z]+|[A-Z]+):\d{6}(/(draft|\d+\.\d+\.\d+))$",
         json_schema_extra={"readOnly": True},
     )
 
     identifier: DANDI = Field(
         title="Dandiset identifier",
         description="A Dandiset identifier that can be resolved by identifiers.org.",
-        pattern=r"^DANDI:\d{6}$",
+        # Vendored:
+        # pattern=r"^DANDI:\d{6}$",
+        # Unvendored:
+        pattern=r"^[A-Z]+:\d{6}$",
```

Comment:
TODO: left a comment on top where we define all of such things.  We might need to see if "IDENTIFIERS_PREFIX" is defined, and if so -- use it as is, and if not defined -- make it into such unvendored regex.